wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/Command.java

```diff
@@ -19,6 +19,10 @@
  * specified explicitly from the command implementation.
  *
  * <p>This class is provided by the NewCommands VendorDep
+ *
+ * @see <a
+ *     href="https://docs.wpilib.org/en/stable/docs/software/commandbased/index.html">Command-Based
+ *     Programming on frc-docs</a>
```

Comment:
Do we need to say "on frc-docs" on all of these?

Reply:
I think it's good to say where the link points to.